core/tests/test_utils.py

```diff
@@ -31,6 +31,7 @@
 import unittest
 
 from constants import constants
+import contextlib2
```

Comment:
This shouldn't be here, since it's not part of our codebase. If it's an external lib it should go with the google.appengine.api stuff; if it's a stdlib it should go with unittest at the top.

Reply:
See isort.cfg in the root folder, you may need to label it correctly so that it's recognized as being third-party.

Reply:
Done.